config/application.yml.default

```diff
@@ -140,6 +140,13 @@ in_person_email_reminder_final_benchmark_in_days: 1
 in_person_email_reminder_late_benchmark_in_days: 4
 in_person_proofing_enabled: false
 in_person_enrollment_validity_in_days: 30
+in_person_enrollments_ready_job_email_body_pattern: '\A\s*(?<enrollment_code>\d{16})\s*\Z'
+in_person_enrollments_ready_job_cron: '0/10 * * * *'
+in_person_enrollments_ready_job_enabled: false
+in_person_enrollments_ready_job_queue_url: ''
+in_person_enrollments_ready_job_max_number_of_messages: 10
+in_person_enrollments_ready_job_visibility_timeout_seconds: 30
```

Comment:
Just wondering how we decided on the time limit for this and the next config?

Reply:
It's normal with SQS to set `wait_time_seconds` high in order to reduce the number of calls and increase the likelihood that we'll receive items from the SQS on each call. Behind the scenes, SQS is also a distributed queue, which means order is only an approximate and not a strict guarantee (unless we use FIFO) & AWS could potentially be fetching the items from multiple machines.

I'm leaving `visibility_timeout_seconds` at the default, but it's a parameter that may be worth adjusting if the processing time is extended at some point. This controls when items will become visible to other workers (i.e. essentially for the case of an unreported processing failure).